opm/grid/GraphOfGrid.cpp

```diff
@@ -29,8 +29,26 @@
 namespace Opm{
 
 template<typename Grid>
-void GraphOfGrid<Grid>::createGraph (const double* transmissibilities)
+void GraphOfGrid<Grid>::createGraph (const double* transmissibilities,
+                                     const Dune::EdgeWeightMethod edgeWeightMethod)
 {
+    // Find the lowest positive transmissibility in the grid.
+    // This includes boundary faces, even though they will not appear in the graph.
+    WeightType logMinTransm = std::numeric_limits<WeightType>::max();
+    if (transmissibilities && edgeWeightMethod==Dune::EdgeWeightMethod::logTransEdgeWgt)
+    {
+        for (int face = 0; face < getGrid().numFaces(); ++face)
+        {
+            WeightType transm = transmissibilities[face];
+            if (transm > 0 && transm < logMinTransm)
+            {
+                logMinTransm = transm;
+            }
+        }
+        assert(logMinTransm < std::numeric_limits<WeightType>::max());
```

Comment:
I also do not expect the assert to ever fail. Therefore, I wanted to keep the code clutter to minimum.

Reply:
I don't think throwing exceptions is cluttering code. We have seen many things happening, and people are creating test cases that might be artificial. So chances are this will happen and maybe to a user of the software.
If the we have compiled with `-DNDEBUG` this will even not be noticed. Otherwise the abort will be rather cryptic when compiled with optimization.

My general rule is to use `assert`. If we are sure that this will never happen (because of how the value should be set elsewhere in our code) and an exception would really hurt performance. Here it seems like we are relying on model making sense and being realistic.